src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -96,6 +97,39 @@ internal static partial class ITypeSymbolExtensions
             return SyntaxFactory.RefType(refKeyword, readOnlyKeyword, underlyingType);
         }
 
+        public static bool TryCreateTypeExpression(SpecialType specialType, [NotNullWhen(true)] out PredefinedTypeSyntax? node)
```

Comment:
Consider making this an extension method, which is more discoverable via completion.